src/Tool/Wrapper/MongoDocumentWrapper.php

```diff
@@ -86,7 +86,7 @@ public function hasValidIdentifier()
     /**
      * {@inheritdoc}
      */
-    public function getIdentifier($single = true)
+    public function getIdentifier($single = true, $flatten = false)
```

Comment:
This class / method is not final, so it is a BC break.

Reply:
Missed that one, thanks